### PR TITLE
Adding V0.1 skirt buttons

### PR DESCRIPTION
**Pull request by Tircown**
Adding V0.1 skirt buttons with 10 common pictograms:
- extrude
- arrow up/down
- heatbed
- hotend
- caselight
- pause/resume
- stop
- park
- onoff
- blank

No hardware required.

Thank you

---------------------

  * [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

### CODE REVIEW

**Comment by cbc02009** [comment]
Do you have any part numbers for switches that users could reference to make sure they have something that fits?

**Comment by Tircown** [comment]
Those are the basic one you have in every arduino starter kits for instance.
Search for "tactile switch 6x6x5mm" (5mm height) or similar products to the Mouser part number 611-PTS645TH502LFS

**Comment by cbc02009** [comment]
> Those are the basic one you have in every arduino starter kits for instance. Search for "tactile switch 6x6x5mm" (5mm height) or similar products to the Mouser part number 611-PTS645TH502LFS

It not necessary I guess. I was just thinking it might help people source the part if they weren't already well versed in jellybean electronic components.

File: printer_mods/README.md [hunk 1]
```diff
@@ -263,6 +263,7 @@ like so:
 |Tircown| [Display mount 5"](./Tircown/Display_mount_5inch) | Tiltable mount for a 5" tactile display | :x: | :x: | :heavy_check_mark: | :x: |
 | | [Raspberry in plug panel](./Tircown/Raspberry_plug_panel) | Raspberry mount vertically in the plug panel, allow easy access to ports  | :x: | :x: | :heavy_check_mark: | :x: |
 | | [DIN mount RS-15-5 PSU](./Tircown/DIN_mount_RS-15-5) | A DIN mount for the RS-15-5  | :grey_question: | :grey_question: | :heavy_check_mark: | :grey_question: |
+| | [V0.1 Skirt buttons](./Tircown/V01_skirt_buttons) | Buttons for V0.1 spec skirts | :heavy_check_mark: | :x: | :x: | :x: |
```
Comment (KiloQubit) [inline printer_mods/README.md:266]: please add an `:x:` for the Trident column

File: printer_mods/Tircown/V01_skirt_buttons/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Voron 0.1 skirt buttons
+
+### Instructions
+For each button you will need to print: 1x *Nut.stl*, 1x *Lock.stl*, 1x *Button-\<name\>.stl*
+1. Print with a 0.1mm layer_height (after the first layer) to improve the quality of the threads,
+2. Solder two wires toward the inside of the legs of a 5mm tactile switch, add some heatshrink,
+3. Push the 5mm tactile switch into the button housing and snap the locking piece behind,
+4. Install the button throught a hexagonal hole in the skirt and screw the printed nut behind,
+5. Optionnal, use the hole in the locking piece and a ziptie for attaching the cables (strain relief).
```
Comment (KiloQubit) [inline printer_mods/Tircown/V01_skirt_buttons/README.md:9]: Replace `Optionnal` with `Optional`

**Review by KiloQubit — CHANGES_REQUESTED** [review]

File: printer_mods/README.md (hunk 1 above)
Comment (Tircown) [reply printer_mods/README.md]: Thank you. fixed

File: printer_mods/Tircown/V01_skirt_buttons/README.md (hunk 2 above)
Comment (Tircown) [reply printer_mods/Tircown/V01_skirt_buttons/README.md]: Thank you. fixed

**Review by cbc02009 — APPROVED** [review]

**Review by KiloQubit — APPROVED** [review]

**Pull request merged**